lightly/api/version_checking.py

```diff
@@ -52,7 +49,7 @@ def do_version_check(current_version: str):
 
 
 def get_versioning_api() -> VersioningApi:
-    configuration = Configuration()
+    configuration = api_workflow_client.get_api_client_configuration_no_token()
```

Comment:
do we really not have the token here already? We should unless this code is called directly, no? Couldn't we rather pass a flag to not raise an exception in `get_api_client_configuration`?

Reply:
> do we really not have the token here already? 

No, we don't have a token.

> Couldn't we rather pass a flag to not raise an exception in get_api_client_configuration?

We could 👍